jira/package.json

```diff
@@ -1,24 +1,25 @@
 {
-  "name": "zendesk-to-dust",
+  "name": "jira-to-dust",
   "version": "1.0.0",
   "description": "Script to import Jira issues to Dust datasources",
   "main": "index.ts",
   "scripts": {
-    "issues": "ts-node jira-issues-to-dust.ts",
```

Comment:
No idea what's wrong with ts-node but it wouldn't run
![Screenshot 2025-04-25 at 19 11 39](https://github.com/user-attachments/assets/895e108b-aed9-42f3-b43e-9b2f6b27dada)

I know nothing about ts but using tsx as recommended [here](https://stackoverflow.com/a/76343394/11160877) solved the issue.